### PR TITLE
Added Cassandra Sink, Heron Adapter, and examples

### PR DESCRIPTION
**Pull request by srkukarni**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -176,6 +185,7 @@ project(':examples-heron') {
         compile project(':core')
         compile project(':aerospike')
         compile project(':twitter')
+		compile project(':cassandra')
```
Comment (aahmed-se) [inline build.gradle]: Fix the space

File: cassandra/src/main/resources/reference.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+cassandra.roots: "127.0.0.1"
```
Comment (aahmed-se) [inline cassandra/src/main/resources/reference.conf]: you can specify at such
cassandra{
  keyspace: "test"
  keyname: "testkey"
  columnfamily: "testfamily"
  columnname: "testcolumn"
}

File: build.gradle (hunk 1 above)
Comment (srkukarni) [reply build.gradle]: Fixed

File: cassandra/src/main/resources/reference.conf (hunk 2 above)
Comment (srkukarni) [reply cassandra/src/main/resources/reference.conf]: Changed

**Review by aahmed-se — APPROVED** [review]

**Pull request merged**